docs/reference/experiment_data.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: default
+title: Experiment data
+parent: Reference
+nav_order: 5
+permalink: /reference/experiment-data/
+---
+
+# Experiment data
+
+This page describes how to obtain the raw data produced during an experiment.
+This includes corpora, crashes and logs from fuzzers.
+It does not include the metrics used to generate a report, that data is linked
+to in the report itself.
+
+This page is written for users of the FuzzBench service. The same concepts apply
+to users running fuzzbench on their own however, the Google Cloud Storage buckets
+will be different.
+
+## Getting the data
+
+[commondatastorage.googleapis.com/fuzzbench-data/](http://commondatastorage.googleapis.com/fuzzbench-data/)
+provides an XML file listing every from every experiment that is available for
+download. It isn't very usable for use in a browser but can probably be used by
```

Comment:
Just copy index.html and icons dir from gs://chromium-browser-asan and then rephrase. we use this everywhere for browsing.